sql/migrations/20200504144725_world.sql

```diff
@@ -0,0 +1,34 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20200504144725');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20200504144725');
+-- Add your query below.
+
+
+-- Alowicious Czervik correct location
+UPDATE `creature` SET `position_x` = 1660.02, `position_y` = -4391.0, `position_z` = 23.2680, `orientation` = 4.771000 WHERE `guid` = 86237;
+
+-- Opus correct location
+UPDATE `creature` SET `position_x` = -7468.840820, `position_y` = -2186.785156, `position_z` = 164.418686, `orientation` = 2.294904, `movement_type` = 0, `wander_distance` = 0 WHERE `guid` = 4567;

```

Comment:
I see several different positions for this npc in sniff, so why are you removing the waypoints? He must be moving.

-7463.3 -2187.77 167.351 2.69095
-7477.56 -2188.48 165.374  3.08992
-7461.74 -2188.44 167.438 2.57953
-7461.38  -2188.6 167.459 2.59347
-7469.25  -2186.32 164.268 2.21123

These are coordinates i have him at.

Reply:
He definitely does not move, and his current position in vmangos is incorrect. Maybe he has different spawn points?
https://youtu.be/5soMRrt-3e4